packages/data/src/components/registry-provider/index.js

```diff
@@ -10,6 +10,22 @@ import defaultRegistry from '../../default-registry';
 
 const { Consumer, Provider } = createContext( defaultRegistry );
 
+/**
+ * A context consumer component which provides to the rendered child function a
+ * reference to the current registry context.
+ *
+ * @link https://reactjs.org/docs/context.html#contextconsumer
```

Comment:
Prefer the `@see` tag ([see](https://github.com/WordPress/gutenberg/issues/14334#issuecomment-481592531))

Reply:
> Prefer the `@see` tag ([see](https://github.com/WordPress/gutenberg/issues/14334))

👍 Updated in c10bcab33